test.sh.in

```diff
@@ -372,7 +395,7 @@ if [ "$RUN_CIT_TEST" == true ] || [ "$RUN_FIT_TEST" == true ] ; then
   # time to make the env instead of doing sleeps...
   setupVirtualEnv
   waitForAPI
-  nodesOn
+  nodesOn &
```

Comment:
can you embed comment in code , to explain the rational.
as you said, some node take long time to boot, and CIT/FIT discovery will wait.
but only you and me knows, so comment in code is a good way to let every reader know.